### PR TITLE
Test/discover test poc

### PR DESCRIPTION
**Pull request by ggilchrist-ledger**
This PR introduces Playwright UI tests for the Discover section of the app, and specifically performing actions in a third party external app to make sure Ledger Live and the SDK handle Live apps properly. 

The tests require the [dummy live app](https://github.com/LedgerHQ/dummy-live-app) to be running on localhost:3001, which happens automatically in the test setup (see `webServer` in `playwright.config.ts`) to ensure the tests work on the CI. The tests then navigate to the discover section and perform javascript functions on the Live app iframe to trigger Nano actions via the Live SDK.

Video demo of tests running locally:

https://user-images.githubusercontent.com/87659842/151358523-f88971ae-62cd-4309-8c5d-e9fc004bf0ca.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "@ledgerhq/hw-transport-http": "6.20.0",
     "@ledgerhq/hw-transport-node-hid-singleton": "6.20.0",
     "@ledgerhq/ledger-core": "6.14.5",
-    "@ledgerhq/live-common": "^21.26.1",
+    "@ledgerhq/live-common": "^21.28.0",
```
Comment (ggilchrist-ledger) [inline package.json:52]: Change before merging!

**Review by valpinkman — APPROVED** [review]

**Comment by valpinkman** [comment]
Great to have new tests on discover section. ✔️ Good for me

**Review by valpinkman — APPROVED** [review]

**Review by alalmi-ledger — COMMENTED** [review]
LGTM

**Pull request merged**